sdk/search/sample.bicepparam

```diff
@@ -0,0 +1,8 @@
+using './test-resources.bicep'
```

Comment:
woah I have not seen this bicepparam file yet - what is it used for?

Reply:
I was having trouble using the `New-TestResources.ps1` script, so this file is a template for contributors to use to deploy test resources.